dts/bindings/haptics/ti,drv2605.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Cirrus Logic, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  DRV2605 Haptic Driver for ERM and LRA with built-in library and smart-loop
+  architecture.
+
+compatible: "ti,drv2605"
+
+include: i2c-device.yaml
+
+properties:
+  ti,actuator-mode:
+    type: string
+    enum:
+      - "ERM Mode"
+      - "LRA Mode"
+  ti,feedback-brake-factor:
+    type: string
+    enum:
+      - "1x"
```

Comment:
Typically string enums are formatted like macros, with no spaces, and shortened as much as possible:
```
- "1X"
- "2X"
...
- "DISABLED"
```
This makes it easy to directly use the value within the drivers by concatenation like:
```
#define TI_FEEDBACK_BRAKE_FACTOR_1X 1
#define TI_FEEDBACK_BRAKE_FACTOR_2X 2
...

uint8_t break_factor = _CONCAT(TI_FEEDBACK_BRAKE_FACTOR_, DT_INST_STRING_TOKEN(inst, ti_feedback_brake_factor))
```

Reply:
ack